CITATION.bib

```diff
@@ -1,6 +1,8 @@
-@misc{Qiskit,
-    author = {{Qiskit contributors}},
-    title = {Qiskit: An Open-source Framework for Quantum Computing},
-    year = {2023},
-    doi = {10.5281/zenodo.2573505}
+@misc{javadiabhari2024quantum,
+      title={Quantum computing with Qiskit},
```

Comment:
```suggestion
      title={Quantum computing with {Q}iskit},
```
we don't care much about how people's style files do the rest of the title, but Qiskit always wants a capital letter